404.html

```diff
@@ -1,44 +1,24 @@
-<!DOCTYPE html>
-<html lang="en" style="font-family: 'Roboto', sans-serif; font-size: 16px; background: #333; color: #FFF;">
-	<head>
-		<link href="favicon.jpeg" rel="icon" type="image/jpg"/>
-		<meta charset="utf-8">
-		<link href="css/style.css" rel="stylesheet">
-		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto&display=swap">
-		<meta name="viewport" content="width=device-width, initial-scale=1.0">
-		<script>
-</script>
-		<meta http-equiv="X-UA-Compatible" content="IE=edge">
-		<meta name="description" content="">
-		<meta name="author" content="">
-		<title>404: THAT TECH SHOW - Hosted by Chris Addams & Sam Gregory</title>
-		<script async src="https://www.googletagmanager.com/gtag/js?id=G-TEM8PPB2T0"></script>
-		<script>
-	if (window.location.href.match(/thattech/)) {
-		window.dataLayer = window.dataLayer || [];
-		function gtag(){dataLayer.push(arguments);}
-		gtag('js', new Date());
+---
+---
 
-		gtag('config', 'G-TEM8PPB2T0');
-	}
-</script>
-	</head>
-	<body style="padding-top: 0; margin: 0;">
-		<main class="main" style="padding-left: 15px; padding-right: 15px;">
-			<header class="header-wrap" data-pgc="tts.header">
-				<a class="header-title" href="/">That Tech Show</a>
-				<a href="https://www.patreon.com/thattechshow" target="_blank" rel="noreferrer nofollow" class="button-link patreon-link"> <svg style="vertical-align: middle;" viewBox="0 0 512 512" width="15" height="15">
-						<path fill="white" d="M512 194.8c0 101.3-82.4 183.8-183.8 183.8-101.7 0-184.4-82.4-184.4-183.8 0-101.6 82.7-184.3 184.4-184.3C429.6 10.5 512 93.2 512 194.8zM0 501.5h90v-491H0v491z"></path>
-					</svg>
-                    Buy us a coffee! </a>
-			</header>
-			<div class="hero-wrapper">
-				<section>
-					<p style="font-size: 20em; margin-bottom: 0; margin-top: 0;">404</p>
-					<h1>Oops, that can't be found</h1>
-					<a href="/" style="color: #fff; text-decoration: none;">Back home</a>
-				</section>
-			</div>
-		</main>
-	</body>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    {% include head.html %}
+  </head>
+  <body style="padding-top: 0; margin: 0">
+    {% include header.html %}
```

Comment:
This conflicts with Pinegrow. We either use liquid or Pinegrow components (which it already was a Pinegrow component)